warrior/Framework/Utils/cli_Utils.py

```diff
@@ -593,8 +594,26 @@ def send_commands_from_testdata(testdatafile, obj_session, **args):
                 result, response = _send_cmd_get_status(new_obj_session, details_dict, index=i, system_name=system_name)
                 result, response = _send_command_retrials(new_obj_session, details_dict, index=i,
                                                           result=result, response=response, system_name=system_name)
-                response_dict = _get_response_dict(details_dict, i, response,
-                                                   response_dict)
+                response_dict, resp_list_item = _get_response_dict(details_dict, i, response,
+                                                   response_dict, resp_list_item)
+                sys_name = ses_name = ''
+                if response_dict.values()[i] is not None:
+                    if details_dict["sys_list"][i] == '' or details_dict["sys_list"][i] is None:
+                        sys_name = system_name
+                    else:
+                        sys_name = details_dict["sys_list"][i]
+                    if details_dict["session_list"][i] == '' or details_dict["session_list"][i] is None:
+                        session_name = args.get("session_name")
+                        ses_name = session_name
+                    else:
+                        ses_name = details_dict["session_list"][i]
+
+                    session_id = data_Utils.get_session_id(sys_name, ses_name) + "_td_response"
+                    for k in resp_list_item:
+                        if k == resp_list_item[i]:
```

Comment:
Do we need this for loop and if condition?? can't we directly use resp_list_item[i] in pNote.